packages/storybook/src/stories/Fonts.tsx

```diff
@@ -26,26 +33,34 @@ export default function Fonts(props: Props) {
           {index > 0 && <hr />}
           <section>
             <div>
-              <h1 style={{ fontFamily: styles.family, fontSize: '4.125rem' }}>{styleKey}</h1>
+              <h1 style={{ fontFamily: styles.family, fontSize: secondary.variant.h2 }}>
+                {styleKey}
+              </h1>
             </div>
             <div>
-              <h2 style={{ marginLeft: '12px' }}>{'Font: ' + fontFormat(styles.family)}</h2>
+              <h2 style={{ marginLeft: '12px', fontFamily: styles.family }}>
```

Comment:
Should this be `.75rem`?

Reply:
Since this is Storybook, `px` is ok but if we want to make it consistent with rem, then it should be `1.2rem`

Reply:
Updated to `rem`